gitau_james/decoder_2by4/decoder_2by4.h

```diff
@@ -0,0 +1,50 @@
+/*
+     #Created on: Nov 15, 2016
+            #Author: GITAU JAMES
+    #Decoder module
+*/
+
+
+#include<systemc.h>
+
+SC_MODULE(decoder){
+	//input and output ports
+	sc_in<bool>d_a,d_b;
+	sc_out<bool>d_c,d_d,d_e,d_f;
```

Comment:
You were supposed to reuse the already existing decoder 1by2 this is not what the homework was about